src/plugins/data/public/search/search_source/search_source.ts

```diff
@@ -187,10 +207,16 @@ export class SearchSource {
     return this.getField(field, false);
   }
 
+  /**
+   * deprecated, don't use
```

Comment:
Could use `@deprecated` annotation here.

```suggestion
   * @deprecated Don't use.
```